WordPress/Classes/Utility/ABTesting/ExPlat.swift

```diff
@@ -37,4 +70,28 @@ class ExPlat: ABTesting {
             return .other(variation)
         }
     }
+
+    private func scheduleRefresh() {
+        if ttl > 0 {
+            scheduleTimer?.invalidate()
+
+            /// Schedule the refresh on a background thread
+            DispatchQueue.global(qos: .background).async { [weak self] in
+                guard let `self` = self else {
+                    return
+                }
+
+                self.scheduleTimer = Timer.scheduledTimer(withTimeInterval: self.ttl, repeats: true) { [weak self] timer in
```

Comment:
I've had a lot of issues with timers in the past, I think you should invalidate and reset when the app goes to the background or comes to the foreground.

Reply:
Good point!

Changed in 629db96b2fd19e90bb70da9e652c421eeb521c7e